Tests/BackendTest/BlockChangeRequestTest.cs

```diff
@@ -83,44 +83,43 @@ public async Task ExchangeRequests_ExchangingRequests_ExchangedRequests()
             blockToChange1.BlockFrom = block1.Clone();
             blockToChange1.BlockTo = block2.Clone();
             blockToChange1.StudentId = student1.Id;
-            await blockChangeService.AddAsync(blockToChange1);
+            await blockChangeService.AddAndFindMatch(blockToChange1);
```

Comment:
refactor this test. It is too long. Add more tests.

Reply:
As I mention before, I dont think that it is good idea split this test - now it is readable but when you split it, it will be hard to read

Reply:
Now it is hard to read. There is a lot of duplicate code...
Rename this test name, so it is clear, that this is a positive test. Add also some negative tests.